src/Acquisition/_Acquisition.c

```diff
@@ -2030,7 +2109,22 @@ init_Acquisition(void)
   AcquisitionCAPI.AQ_Inner = capi_aq_inner;
   AcquisitionCAPI.AQ_Chain = capi_aq_chain;
 
-  api = PyCObject_FromVoidPtr(&AcquisitionCAPI, NULL);
+  api = PyCapsule_New(&AcquisitionCAPI, "AcquisitionCAPI", NULL);
```

Comment:
Ok, ignore this comment. There's something weird going on with my local environment, which I'll sort out later. Probably tox/buildout issues with pulling in the right Acquisition branch.